nengo_loihi/loihi_cx.py

```diff
@@ -1164,12 +1166,12 @@ def receive(self, t, x):
 class ChipReceiveNode(nengo.Node):
     """For receiving host->chip messages"""
 
-    def __init__(self, dimensions, size_out):
+    def __init__(self, dimensions, size_out, **kwargs):
```

Comment:
I think `**kwargs` is confusing here; for me, I don't really remember what other parameters there are to `Node`. If I look it up and see `size_in`, I'll get a `TypeError` if I try to specify it. I think we should bite the bullet and expose `label` and `seed` (or just `label` if seeds are irrelevant here).